lib/content-services/src/lib/i18n/en.json

```diff
@@ -465,6 +465,9 @@
       "ARIA_LABEL": "Toggle {{ name }}"
     }
   },
+  "ADF-TREE": {
+    "LOAD-MORE-BUTTON": "Load more {{ name }}"
```

Comment:
can we probably make it generic and say "Load more entries" so that many other apps don't need extra efforts providing content here? in that case we don't need suffix property on the component level, less tests, etc.

Reply:
Maybe I could add "entries" as default value of the suffix? Then no extra effort would be required but there would be option to customize the tree if someone likes it?

Reply:
I suggest we don't predict the future and don't build options that nobody asked for yet, we could always add later

Reply:
But in our case the customization was already requested, Chapman wants to have `subcategories` suffix there, that's why I added this in the first place